tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -1260,7 +1260,7 @@ STATIC_ASSERT(__cpp_lib_memory_resource == 201603L);
 #endif
 #endif
 
-#if _HAS_CXX20
+#if _HAS_CXX20 && defined(__cpp_lib_concepts) // TRANSITION, GH-395

```

Comment:
I recently asked for a `!defined(__EDG__) // TRANSITION, EDG Concepts support` in this exact same case. I don't really care _which_ we use, but I'd very much like to use one or the other consistently. If you like this style, could you please replace occurrences of the other?

Reply:
Sure, this PR isn't doing anything else, so I can add such a cleanup.

Reply:
Done, as a special exception to our usual "don't mix cleanups and fixes" policy (since the "fix" here is extremely minor, almost cosmetic).